spec/controllers/admin/contests_controller_spec.rb

```diff
@@ -40,7 +40,7 @@
     end
 
     describe "#create" do
-      let(:contest_params) { attributes_for(:contest) }
+      let(:contest_params) { build(:contest).attributes }
```

Comment:
I didn't know you can do this trick to get a hash with the IDs of the relations. Nice one!